rca/project_styleguide/templates/patterns/organisms/facilities/facilities.html

```diff
@@ -1,21 +1,28 @@
 <div class="section__row grid facilities">
     <h2 class="facilities__heading heading heading--four">{{ title }}</h2>
-    <div class="facilities__introduction">
-        <p class="introduction">{{ introduction }}</p>
-        <a href="{{ link }}" class="facilities__view-all facilities__view-all--medium body body--one link link--primary link--link">
-            <span class="link__label">{{ link_text }}</span>
-            <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
-        </a>
-    </div>
-    <div class="facilities__action">
-        <a href="{{ link }}" class="facilities__view-all body body--one link link--primary link--link">
-            <span class="link__label">{{ link_text }}</span>
-            <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
-        </a>
-    </div>
-    <div class="facilities__body">
-        <p>{{ body }}</p>
-    </div>
+    {% with facilities_snippet=page.facilities_snippet %}
+        <div class="facilities__introduction">
+            {% if facilities_snippet.introduction %}
+                <p class="introduction">{{ facilities_snippet.introduction }}</p>
+            {% endif %}
+
+            <a href="{{ link }}" class="facilities__view-all facilities__view-all--medium body body--one link link--primary link--link">
+                <span class="link__label">{{ link_text }}</span>
+                <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
+            </a>
+        </div>
+        <div class="facilities__action">
+            <a href="{{ link }}" class="facilities__view-all body body--one link link--primary link--link">
+                <span class="link__label">{{ link_text }}</span>
+                <svg width="12" height="8" class="link__icon"><use xlink:href="#arrow"></use></svg>
+            </a>
+        </div>
+        {% if facilities_snippet.copy %}
+        <div class="facilities__body">
+            <p>{{ facilities_snippet.copy }}</p>
```

Comment:
Do you want to respect the new lines specified by the user? If you do, then:
`{{ facilities_snippet.copy }}` -> `{{ facilities_snippet.copy|linebreaks }}`

https://docs.djangoproject.com/en/2.2/ref/templates/builtins/#linebreaks